app/src/stores/pomodoro/constants.ts

```diff
@@ -2,7 +2,7 @@ import type { StateType } from './types'
 
 export const DEFAULT_STATE_VALUES: StateType = {
   id: null,
-  counterContent: '25:00',
+  counterContent: '',
```

Comment:
What's the reason for this change? The store shouldn't change just because we are adding animations, no? :thinking:

Reply:
After reloading the page, there's a small delay where the animation changes to the default counter content, so I tried setting it to an empty string and I haven't seen any problem yet, but this is something we should check better.

Reply:
@ehdlg Can you look into this?
I think the animation shouldn't dictate to have an empty `counterContent` by default :thinking: 
The rest of the PR looks fine to me :+1:

Reply:
We agreed that this will be worked in another PR in the future.